### ISSUE TITLE
Notifications on Oficial Mobile App

### ISSUE THREAD

**Issue by mastermailgeral** [sendingtk/chatwoot#100]
### Describe the bug

With last version i am unable to receive the push notifications on oficial mobile app from chatwoot. I am using the IOS app.

### To Reproduce

Ubuntu 20.04 VPS on Hetzner , Self hosted Chatwoot (3.12.0) with docker 23.0 and portainer 2.20.1 , Evolution API 1.8.2

### Expected behavior

When message are received on Chatwood, the mobile app should alert with push notification. Email notifications are working.

### Environment

Docker

### Cloud Provider

None

### Platform

Mobile

### Operating system

Ubuntu 20.04

**Comment by mastermailgeral** [comment]
I have instaled the chatwoot 3.12.6 without docker and still dont receive the push notifications on oficial mobile app. Yesterday i test the official version of chatwoot 3.12.0 and worked, i think this can be some type of configuration in this version of Chatwoot :(

**Comment by tonnybarros** [comment]
Have you tested it on another cell phone? Could it be a notification from the blocked cell phone?

**Comment by mastermailgeral** [comment]
Yes i tried and the problem presist. @tonnybarros do you have this version working with push notification using the oficial mobile app of chatwoot?

**Comment by mastermailgeral** [comment]
So i try again install the latest version and nothing. I realy think that is something about with this fork and a really issue. I explain, i had installed the official version of chatwood with same docker, same configuration, same database. Works perfectly, i delete de original stack, delete the database and simply istall this latest version 3.12.6 with new database and exactly the same variables and the problem appears and persist.

**Comment by mastermailgeral** [comment]
Thanks @nestordavalos for your clarification, it makes sense and is exactly the answer I needed. Let me just say thank you for the excellent version of chatwoot you have developed. One last question, do you know of a simpler way to implement the notifications, maybe another app?

**Comment by matisidler** [comment]
Hey @mastermailgeral! Have you found a workaround for this?

**Comment by mastermailgeral** [comment]
> Hey @mastermailgeral! Have you found a workaround for this?

Nop, like @nestordavalos are reported for this version of chatwoot you cant have the notifications on ios app in normal way.
